mtools/mloginfo/sections/query_section.py

```diff
@@ -55,7 +55,7 @@ def run(self):
                     progress_curr = self.mloginfo._datetime_to_epoch(le.datetime)
                     self.mloginfo.update_progress(float(progress_curr-progress_start) / progress_total)
 
-            if le.operation in ['query', 'getmore', 'update', 'remove'] or le.command in ['count', 'findandmodify', 'geonear']:
+            if le.operation in ['query', 'getmore', 'update', 'remove'] or le.command in ['count', 'findandmodify', 'geonear','find']:
```

Comment:
Small styling issue: could you please add a space after the comma so it looks like the other entries in the array?